CHANGELOG.md

```diff
@@ -16,6 +16,19 @@ of response and not the raw API endpoint response
 
 ### Added
 
+- **CUMULUS-2575**
+  - Adds `POST /granules` API endpoint to create a granule
+  - Updates `@cumulus/db/src/model/granules` functions `get` and `exists` to
+    enforce parameter checking so that requests include either (granule\_id
+    and collection\_cumulus\_id) or (cumulus\_id) to prevent incorrect results.
+  - `@cumulus/message/src/Collections.deconstructCollectionId` has been
+    modified to throw a descriptive error if the input `collectionId` is
+    undefined rather than `TypeError: Cannot read property 'split' of
+    undefined`. This function has also been updated to throw descriptive errors
+    if an incorrectly formated collectionId is input.
```

Comment:
These two notes should be under the "Changed" header

Reply:
82d8c679fc